src/resources/profiles.jsx

```diff
@@ -0,0 +1,82 @@
+import { 
+    List, 
+    TextField, 
+    EmailField,
+    DateField,
+    Datagrid,
+    Edit,
+    SimpleForm,
+    TextInput,
+    DateTimeInput,
+    RadioButtonGroupInput,
+    SelectField,
+    Create,
+    Show,
+    SimpleShowLayout
+} from "react-admin";
+
+const profileFilters = [
+  <TextInput source="q" label="Search" alwaysOn />,
+];
+
+const gender = [
+  { id: 0, name: "male" },
+  { id: 1, name: "female" },
+  { id: 2, name: "divers" },
+];
+
+export const ProfileList = () => (
+  <List filters={profileFilters}>
+    <Datagrid rowClick="show">
+      <TextField source="id" />
+      <TextField source="username" />
+      <EmailField source="email" />
```

Comment:
we could also show the full name here. Fyi there's `FunctionField` for showing them in one.
```
<FunctionField
    label="Name"
    render={record => `${record.surname} ${record.name}`}
/>;
```

Reply:
I'll add it